pkgs/applications/virtualization/docker/buildx.nix

```diff
@@ -3,6 +3,7 @@
 buildGoModule rec {
   pname = "docker-buildx";
   version = "0.8.2";
+  gitCommit = "6224def4dd2c3d347eee19db595348c50d7cb491";
```

Comment:
Is this going to be automatically bumped? See: https://github.com/NixOS/nixpkgs/pull/167704/files.

Reply:
No it's not. I haven't found a good way to get the commit sha in a better way. But I am fairly new to nix so maybe there is one?

Reply:
I removed the hardcoded `gitCommit` field, just showing the release version would already be an improvement for me (and hopefully others). This way there shouldn't be any issues with automatic upgrades.